### PR TITLE
feat: allow custom html

### PR DESCRIPTION
**Pull request by thepassle**
closes https://github.com/LarsDenBakker/web-test-runner/issues/7

### CODE REVIEW

File: src/core/TestRunnerConfig.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export interface TestRunnerConfig {
   server: Server;
   address: string;
   port: number;
+  html?: string;
```
Comment (LarsDenBakker) [inline src/core/TestRunnerConfig.ts:11]:
```suggestion
  testRunnerHtml?: string;
```

Or something else less generic?

File: demo/customhtml.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  html: `
```
Comment (LarsDenBakker) [inline demo/customhtml.config.js:2]: Should we make this a function that receives the config? That way, people can inject the test framework specified in the config.

Comment (thepassle) [reply demo/customhtml.config.js:2]: Yeah was contemplating something like that as well. Will update later 🙂

Comment (thepassle) [reply demo/customhtml.config.js]: ptal

**Comment by LarsDenBakker** [comment]
👍

**Pull request merged**